packages/server/postgres/extensions/pgvector/Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: all clone build install
```

Comment:
instead of building it locally, whatcha think about just using an image with pgvector already installed? https://github.com/ParabolInc/parabol/blob/ec5d15bd851430cc0856a1455ee8c741ec40b26a/.github/workflows/build.yml#L17

Reply:
Why clone the pgvector repo?

Reply:
Yeah, this is cleaner. Originally when I wrote the Makefile I was worried there might be some additional steps to get the extension working. I'll inline it instead